FhirToDataLake/native/parquet/cpp/src/ParquetWriter.cpp

```diff
@@ -14,7 +14,7 @@ int WriteToParquet(const shared_ptr<arrow::Table> table, byte** outputData, int*
     if (!status.ok())
     {
         string errorDetail = status.ToString();
-        strncpy(errorMessage, errorDetail.c_str(), 200);
+        WriteErrorMessage(errorDetail, errorMessage);
```

Comment:
why not initilize _readOptions in `ParquetWriter(const unordered_map<string, string>& schemaData)`?